templates/datafy-agent-pre-uninstall-k8s.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and .Values.ebsCsiProxy.enabled (ne (include "datafy-agent.agentModeNormalized" .) "sensor") }}
+{{- $ctx := . }}
+{{- range $hook := list "pre-delete" }}
```

Comment:
dont copy paste and leave a list of 1 variable - its redundant.
inline the `pre-delete`